plugins/modules/lxd_container.py

```diff
@@ -220,9 +222,9 @@
         source:
           type: image
           mode: pull
-          server: https://images.linuxcontainers.org
-          protocol: lxd # if you get a 404, try setting protocol: simplestreams
-          alias: ubuntu/xenial/amd64
+          server: https://cloud-images.ubuntu.com/releases/
+          protocol: simplestreams
+          alias: 22.04
```

Comment:
`22.04` is a floating point number, not a string. Use quotes here:
```suggestion
          alias: "22.04"
```

Reply:
true, I shouldn't just have hacked this in the Github UI